test/danetest.c

```diff
@@ -192,7 +190,7 @@ static ossl_ssize_t hexdecode(const char *in, void *result)
     for (byte = 0; *in; ++in) {
         int x;
 
-        if (isspace(_UC(*in)))
+        if (isspace(*in))
```

Comment:
> The _UC() should be removed for ossl_isXXX calls.

I'd say they *can* be removed, not *should*. And they can thanks to c >= 0 check. Or maybe you meant to write _UC should be removed, because it's non-compliant? This might imply that it should be *replaced* with something else [in cases when actually needed like with iswhatever()].